app-editors/gnome-latex/gnome-latex-3.42.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2
+
+DESCRIPTION="Integrated LaTeX environment for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/GNOME-LaTeX https://gitlab.gnome.org/swilmet/gnome-latex"
```

Comment:
I've added remote-id to all GNOME packages' `metadata.xml` files, so let's drop the gitlab URL from `HOMEPAGE`.